src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1674,4 +1674,19 @@
   <data name="ReplaceStringLiteralWithCharLiteralCodeActionTitle" xml:space="preserve">
     <value>Replace string literal with char literal</value>
   </data>
+  <data name="DetectPreviewFeaturesDescription" xml:space="preserve">
+    <value>An assembly has to opt into preview features before using them.</value>
+  </data>
+  <data name="DetectPreviewFeaturesMessage" xml:space="preserve">
+    <value>Using '{0}' requires opting into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
```

Comment:
Not sure if the link should be dropped in favor of the help link (which will contain the rule doc at some point)

Reply:
Good catch! This is still being discussed. It's minor enough that I'm fine with it going in as is and once the docs team decided which way to go, I'll change the error message